code/game/gamemodes/wizard/artefact.dm

```diff
@@ -72,6 +72,14 @@
 	activate_descriptor = "hunger"
 	rend_desc = "Reverberates with the sound of ten thousand moos."
 
+/obj/item/weapon/veilrender/lesserveilrender
+	name = "lesser veil render"
+	desc = "A wicked curved blade of alien origin, recovered from the ruins of ancient temple."
```

Comment:
"recovered from the ruins of ancient temple." there should be an "an" between "of" and "ancient"